waku/protocol/v2/waku_relay.nim

```diff
@@ -74,6 +74,14 @@ method unsubscribe*(w: WakuRelay,
   else:
     await procCall FloodSub(w).unsubscribe(topics)
 
+method unsubscribeAll*(w: WakuRelay,
+                    pubSubTopic: string) {.async.} =
```

Comment:
Formatting, either indent to same column or bring up on same row

Reply:
Fixed in 4979718